VOKMockUrlProtocol.m

```diff
@@ -155,6 +166,10 @@ - (NSArray *)resourceNames
                 [resourceNames addObject:[[name stringByAppendingFormat:AppendSeparatorFormat, bodyString] mutableCopy]];
             }
             [resourceNames addObject:[[name stringByAppendingFormat:AppendSeparatorFormat, bodyHash] mutableCopy]];
+            
+            if (AllowsWildcard) {
+                [resourceNames addObject:[[name stringByAppendingFormat:AppendSeparatorFormat, Wildcard] mutableCopy]];
+            }
         }
     }
```

Comment:
I think that'd be better... but I'm also wondering if the logic for assembling the various possible resource names might be simplified by, rather than building up the resource name strings as we go along, building up arrays of the possible values for each part, then assembling all possible combinations at the end, and inserting wildcards then as appropriate.